packages/victory-brush-container/src/victory-brush-container.js

```diff
@@ -83,27 +83,46 @@ export const brushContainerMixin = (base) => class VictoryBrushContainer extends
     const { x, y } = coordinates;
     const { brushStyle, brushComponent, name } = props;
     const brushComponentStyle = brushComponent.props && brushComponent.props.style;
+    const cursor = !props.allowDrag && !props.allowResize ? "auto" : "move";
     return x[0] !== x[1] && y[0] !== y[1] ?
       React.cloneElement(brushComponent, {
         key: `${name}-brush`,
         width: Math.abs(x[1] - x[0]) || 1,
         height: Math.abs(y[1] - y[0]) || 1,
         x: Math.min(x[0], x[1]),
         y: Math.min(y[0], y[1]),
-        cursor: "move",
+        cursor,
         style: defaults({}, brushComponentStyle, brushStyle)
       }) : null;
   }
 
+  getCursorPointers(props) {
+    const cursors = {
+      "yProps": "ns-resize",
+      "xProps": "ew-resize"
+    };
+    if (!props.allowResize && props.allowDrag) {
+      cursors.xProps = "move";
+      cursors.yProps = "auto";
```

Comment:
I still think these should both be "move". If you disagree can you explain why not?

Reply:
yeah, I agree, because if you can't resize, but you can move, makes sense